### DISCUSSION TITLE
请问如何只在关注列表中参与天选时刻和红包抽奖

### DISCUSSION THREAD

**Discussion by Twilight-9** [andywang425/BLTH#242] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/62452364/162375153-372981ac-48c1-4952-8635-30953f824f1d.png)
天选时刻仅勾选从已关注列表参与天选时刻
![image](https://user-images.githubusercontent.com/62452364/162375278-a9614a71-e4b9-4f12-bba5-83de473918c6.png)
红包这里也仅勾选从已关注列表获取红包抽奖。
但是！他会自动关注其他up，然后消息里就不停地有新消息。我只是想从已关注列表里启用天选时刻和红包抽奖，没有办法吗？

**Comment by andywang425** [comment, marked as answer]
请仔细阅读一下awpush后小问号内的内容：
> 启用这个功能后你的部分设置将会由服务端决定，如【天选时刻数据获取方式】，检查房间最大数量，请求间隔等。

关闭awpush即可。
